packages/block-library/src/column/index.js

```diff
@@ -21,7 +21,6 @@ export const settings = {
 	icon,
 	description: __( 'A single column within a columns block.' ),
 	supports: {
-		inserter: false,
```

Comment:
I have a similar PR, I believe we need a notice in the UI somewhere when we reach the limit of max 6 columns. It becomes more of a guideline than a requirement.

Reply:
I merged your PR to this :)

On mobile side we temporary override `support: inserter` to true in `index.native.js` and will remove it after your PR gets merged.

On mobile Columns works slight different because we are wrapping them depending on screen size. It do not visualise the layout in 100% but in the other hand we don't need to worry about the max limit.

I will post you a GIF tomorrow to show what I mean

Reply:
Below GIFs should visualise what I mean.

**Vertical position**
![column vertical](https://user-images.githubusercontent.com/21242757/78976453-7aebeb80-7b16-11ea-9be6-79534e0edc73.gif)

**Horizontal position**
![column vert](https://user-images.githubusercontent.com/21242757/78976435-7293b080-7b16-11ea-9f8b-64fd3a4447cf.gif)

---------
**Few side notes**

Currently our guidelines according to looks like that: [see this comment](https://github.com/wordpress-mobile/gutenberg-mobile/issues/1430#issuecomment-541180270). However we also have hardcoded limitation on block container max-width set to **580px**. The conclusion is that at this moment we should never have more that 2 Columns in the row on mobile. The limitation notice in Columns on mobile will have more sense if we can go above that width limit ( also above **768px**). Then the Columns logic will try to render all Columns in one row. Still we can adjust to add some min-width limit from which we start to wrap in the same manner again. Maybe someday in the future we extend this max-width - don't know exact plan on this but on iPad it feels quite awkward because of lot of empty space on sides - specially in Horizontal layout. I know it's UX to increase readability but not sure if it wasn't also some workaround because of implementation. Please see below screen to see how it looks on iPad.

Apart of above if we have UI notice on web we also should have it on mobile to notice user that layout may be different on web. I'am fully agree with that. @iamthomasbishop do you have any thoughts how this notice should looks like ?

![image](https://user-images.githubusercontent.com/21242757/78977749-ef278e80-7b18-11ea-93cd-09d7b1ebde55.png)